packer/nomad-server/scripts/download-files.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -x
+
+set -e
+
+pushd /tmp/files
+wget \
+	--no-verbose \
+	--input-file=urls \
+	--no-clobber
+
+curl -L -o cni-plugins.tgz "https://github.com/containernetworking/plugins/releases/download/v0.9.0/cni-plugins-linux-$( [ $(uname -m) = aarch64 ] && echo arm64 || echo amd64)"-v0.9.0.tgz
```

Comment:
Also, why are you using `wget` _and_ `curl`? Why not just standardize on one?

`curl` has the `-K/--config` option that allows you to pass in a file of URLs to download, but the format is richer than what `wget` allows, and includes the ability to add comments, so you can describe what the files are that you're downloading are for (in particular, why the versions are so dramatically different in these two AMIs we're building... is that intentional? Should we be installing the same versions everywhere (and thus sharing a common file)?)

Reply:
This is all stuff that, per colin:
" Colin O'Brien:spiral_calendar_pad:  11:32 AM
FWIW the packer stuff is only semi-working. I took it from some project that was super super out of date, and then I worked to update it. Nate's gonna fix it up (next week), since he's used Nomad/Consul before. (edited) 
"

Given that: I don't think it's worth spending the time to make this perfect